### PR TITLE
fix issue with saving hotspots in topic image editor

### PR DESCRIPTION
**Pull request by StianHaugland1**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ services:
       - MYSQL_DATABASE=wordpress
       - MYSQL_USER=wordpress
       - MYSQL_PASSWORD=wordpress
+
+    ports:
+      - 3306:3306
```
Comment (boyum) [inline docker-compose.yml:12]: What does this change entail?

File: h5p-editor-bildetema-words-topic-image/src/h5p/H5PWrapper.tsx [hunk 2]
```diff
@@ -159,7 +159,6 @@ export class H5PWrapper extends H5PWidget<Field, Params> implements IH5PWidget {
   }
 
   private setValueForField = (params: Params): void => {
-    // @ts-expect-error The first element for some reason becomes an empty string. Therefore, we need to put something in front which will be substituted.
-    this.setValue(this.field, [false, ...params]);
+    this.setValue(this.field, [ ...params]);
```
Comment (boyum) [inline h5p-editor-bildetema-words-topic-image/src/h5p/H5PWrapper.tsx:162]: Not sure if we need to copy the array anymore.

```suggestion
    this.setValue(this.field, params);
```

File: h5p-editor-bildetema-words-topic-image/src/h5p/H5PWrapper.tsx [hunk 3]
```diff
@@ -159,7 +159,6 @@ export class H5PWrapper extends H5PWidget<Field, Params> implements IH5PWidget {
   }
 
   private setValueForField = (params: Params): void => {
-    // @ts-expect-error The first element for some reason becomes an empty string. Therefore, we need to put something in front which will be substituted.
-    this.setValue(this.field, [false, ...params]);
+    this.setValue(this.field, [...params]);
```
Comment (boyum) [inline h5p-editor-bildetema-words-topic-image/src/h5p/H5PWrapper.tsx:162]: Not sure if we need to copy the array anymore.

```suggestion
    this.setValue(this.field, params);
```

File: docker-compose.yml (hunk 1 above)
Comment (StianHaugland1) [reply docker-compose.yml:12]: This just exposes the database for local development

**Review by boyum — APPROVED** [review]

**Pull request merged**